config/locales/blacklight.ar.yml

```diff
@@ -100,7 +104,11 @@ ar:
         format: 'الصيغة: %{value}'
         language: 'اللغة: %{value}'
         subject:
```

Comment:
Relevant keys in English: https://github.com/projectblacklight/blacklight/blob/f401bdf1966af5d7e33f5f0e89e6b3665af42b95/config/locales/blacklight.en.yml#L90-L106